test/unit/TestNXOSDriver.py

```diff
@@ -71,10 +73,12 @@ def show(self, command, fmat = 'xml', text = False):
             extension = 'txt'
 
         filename = re.sub(r'[\[\]\*\^\+\s\|\/]', '_', command)
-        mock_file = 'nxos/mock_data/{filename}.{extension}'.format(
+        mock_file = '{filename}.{extension}'.format(
             filename  = filename[0:150],
             extension = extension
         )
+        curr_dir = os.path.dirname(os.path.abspath(inspect.getfile(inspect.currentframe())))
```

Comment:
Switching to __file__, it works just as well and is simpler.  Not bothering to look for benefits for the other approach because simplicity wins.